dd-opentracing-cpp/ubuntu_16_04/0.2.1/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:16.04
```

Comment:
With this kind of directory structure, the tag being produced is `ubuntu-18.04-v0.2.1` (lacking any mention of dd-opentracing-cpp).
Is that OK? Is the fix for it using a different directory structure or changing the rule/config that generates the tag?

Reply:
I don't think there *is* an automated link between directory structure and tag. I did mess up the tags when I manually tagged them though (because I'm so used to the normal way).